src/Lykke.Service.SwiftWithdrawal/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM microsoft/dotnet:2.1-aspnetcore-runtime
```

Comment:
should be FROM microsoft/dotnet:2.1.0-aspnetcore-runtime

Reply:
it seems to me that now microsoft/dotnet:2.1 is the same (the latest version), it is written here https://hub.docker.com/r/microsoft/dotnet/